patches/chrome-browser-resources-settings-about_page-about_page.html.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/settings/about_page/about_page.html b/chrome/browser/resources/settings/about_page/about_page.html
+index 0b9361e77d8988a3388843d392959c761f3c1901..04ed75854e071287b9828f0fc66d3535a678bc58 100644
+--- a/chrome/browser/resources/settings/about_page/about_page.html
++++ b/chrome/browser/resources/settings/about_page/about_page.html
+@@ -131,7 +131,7 @@
+                   </div>
+                   <a hidden$="[[!shouldShowLearnMoreLink_(
+                       currentUpdateStatusEvent_)]]" target="_blank"
+-                      href="https://support.google.com/chrome?p=update_error">
```

Comment:
@bbondy maybe we can handle these in the network delegate by looking at the referrer page?

Reply:
Yep,  I'll try network delegate

Reply:
@bridiver I tested with network delegate and it works well.
With this approach, we don't need to make additional patch file.
However, I think there are some drawbacks with this this approach. WDYT?
* Status bubble shows google's url instead of our url.
* Difficult to catch upstream url change

Reply:
oh, it actually displays the url? Maybe just stick with the patch for now then

Reply:
Yep, when mouse hovers over the link, bubble shows google url.